worker/src/data.ts

```diff
@@ -4,6 +4,21 @@ export const KV_PREFIX_HISTORY = "history";
 export const KV_PREFIX_UUID = "uuid";
 export const KV_MAX_PROCESS_ENTRIES = 850;
 
+export interface Metadata {
+  created: number;
+  updated: number;
+  version: string;
+  installation_type: string;
+  country?: string;
+  extra: boolean;
```

Comment:
Instead of a boolean, we should include what extra things are stored.

Reply:
It does not matter, for everything else we need the full value

Reply:
For now we process everything at once, but that might not always be the case. For example, if we are doing something where we just need to know integrations, it will be good to know if the value contains integrations or not.